shared/ios/Keybase/AppDelegate.m

```diff
@@ -163,18 +163,6 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
       NSLog(@"Remote notification handle finished...");
     });
   } else if (type != nil && [type isEqualToString:@"chat.newmessage"]) {
-    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^(void){
```

Comment:
does android have an equivalent we should remove as well?

Reply:
No